### PR TITLE
[New] Graphql Apollo: An Introduction with Examples

### PR DESCRIPTION
**Pull request by leslitagordita**

### CODE REVIEW

File: docs/guides/development/graphql/graphql-apollo-an-introduction/index.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+      }
+{{< /output >}}
+
+The result of such a query might be
+
+{{< output >}}
+{
+  "data": {
+	"realtor": {
+  	"name": "Terry Musa"
```
Comment (claird) [inline docs/guides/development/graphql/graphql-apollo-an-introduction/index.md:41]: I usually indent two more spaces on this line, the `name` one.

File: docs/guides/development/graphql/graphql-apollo-an-introduction/index.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+modified_by:
+  name: Linode
+title: "Graphql Apollo: An Introduction"
+h1_title: "Graphql Apollo: An Introduction with Examples"
+enable_h1: true
+contributor:
+  name: Cameron Laird
+---
+
+Do you retrieve data from multiple on-line sources to such limited-bandwidth [edge devices](https://www.intel.com/content/www/us/en/edge-computing/edge-devices.html) as smartwatches, mobile telephone handsets, and [Internet-of-things](https://www.linode.com/docs/guides/development/iot/) (IoT) machines?  If so, you want to learn [GraphQL](https://graphql.org/), a maturing standard *query language* for accessing on-line data that particularly fits such applications.  GraphQL began as an internal project at Facebook in 2012.  After a public release in 2015, GraphQL was provided a permanent home with the [GraphQL Foundation](https://graphql.org/foundation/) in 2019. The GraphQL Foundation is under the umbrella of the open-source-oriented [Linux Foundation](https://linuxfoundation.org/).
```
Comment (claird) [inline docs/guides/development/graphql/graphql-apollo-an-introduction/index.md:20]: 2018:  please change `... in 2019. The GraphQL Foundation ...` to `... in 2018. The GraphQL Foundation ...`

Comment (claird) [reply docs/guides/development/graphql/graphql-apollo-an-introduction/index.md:20]: I actually prefer, `After its public release in 2015, the [GraphQL Foundation](https://graphql.org/foundation/) was created in 2018 to provide a permanent home for GraphQL.`

Comment (claird) [reply docs/guides/development/graphql/graphql-apollo-an-introduction/index.md:20]: WOW, is this making me feel rusty:  I'm accustomed to seeing source code and documentation through Git.  I need a little practice viewing **content** this way.

File: docs/guides/development/graphql/graphql-apollo-an-introduction/index.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+} from "@apollo/client";
+
+const cache = new InMemoryCache()
+const uri = 'https://api.spacex.land/graphql/';
+const query = gql`
+  query {
+    launchesPast(limit: 10) {
+      mission_name
+    }
+  }
```
Comment (claird) [inline docs/guides/development/graphql/graphql-apollo-an-introduction/index.md:88]: This line is still missing the backtick I mentioned earlier.  The line should be

    }`

File: docs/guides/development/graphql/graphql-apollo-an-introduction/index.md [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+    npm install graphql @apollo/client cross-fetch
+
+This should take only a few seconds to complete.
+
+Create `client-example.mjs` with the following content:
+
+{{< file "client-example.mjs" >}}
+// cross-fetch helps ensure correct behavior whatever environment
+// needs the HttpLink.
+import fetch from 'cross-fetch';
```
Comment (claird) [inline docs/guides/development/graphql/graphql-apollo-an-introduction/index.md:76]: For no reason I can reconstruct now, the source terminates with semicolons inconsistently.  My current recommendation:  let's remove 'em all.

I'm uncertain how I can best help.  For a change such as this, should I file a counter-**pull request**, or create a new branch, or help you update _this_ one, or ...?

File: docs/guides/development/graphql/graphql-apollo-an-introduction/index.md [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+A minimal implementation based on Apollo’s server-side code is just as quick as the client example above.  Start with
+
+    npm install apollo-server-express express graphql
+
+Create `hello-server.js` with content
+
+{{< file "hello-server.js" >}}
+var {graphql, buildSchema} = require('graphql');
+
+var schema = buildSchema(`
```
Comment (claird) [inline docs/guides/development/graphql/graphql-apollo-an-introduction/index.md:128]: The backticks seem to confuse GitHub's rendering.  On my screen, the type switches to blue rather surprisingly.  It doesn't seem to hurt functionality, though.

Again, I'm ready to remove all the terminal semicolons.

File: docs/guides/development/graphql/graphql-apollo-an-introduction/index.md (hunk 3 above)
Comment (Sachin-Suresh) [reply docs/guides/development/graphql/graphql-apollo-an-introduction/index.md:88]: Hello @claird, thanks for your comments. I have added the missing backtick you mentioned. When I run the `node client-example.mjs` file, I get the following error:

**Comment by Sachin-Suresh** [comment]
And @claird, this is the error when I run the `sudo node server-example.js`:

`exports.AggregateError = globalThis.AggregateError;
                         ^

ReferenceError: globalThis is not defined
    at Object.<anonymous> (/home/sach7411/GraphQL/node_modules/@graphql-tools/utils/index.js:887:26)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/sach7411/GraphQL/node_modules/@graphql-tools/mock/index.js:9:15)
    at Module._compile (module.js:652:30)`

**Comment by Sachin-Suresh** [comment]
> And @claird, this is the error when I run the `sudo node server-example.js`:
> 
> `exports.AggregateError = globalThis.AggregateError;
> ^
> 
> ReferenceError: globalThis is not defined
> at Object. (/home/sach7411/GraphQL/node_modules/@graphql-tools/utils/index.js:887:26)
> at Module._compile (module.js:652:30)
> at Object.Module._extensions..js (module.js:663:10)
> at Module.load (module.js:565:32)
> at tryModuleLoad (module.js:505:12)
> at Function.Module._load (module.js:497:3)
> at Module.require (module.js:596:17)
> at require (internal/module.js:11:18)
> at Object. (/home/sach7411/GraphQL/node_modules/@graphql-tools/mock/index.js:9:15)
> at Module._compile (module.js:652:30)`

**Comment by Sachin-Suresh** [comment]
@claird - Awaiting your response on this please!

**Pull request closed without merging**